vulnfeeds/cves/versions.go

```diff
@@ -256,3 +272,29 @@ func CPEs(cve CVEItem) []string {
 	}
 	return cpes
 }
+
+// Parse a well-formed CPE string into a struct.
+func ParseCPE(cpe_str string) (*CPE, error) {
+	if !strings.HasPrefix(cpe_str, "cpe:") {
+		return nil, errors.New("Invalid CPE string")
+	}
+	if !strings.Contains(cpe_str, ":") {
```

Comment:
This check is redundant, since you already checked the string has `:` in `cpe:`

Reply:
Good catch, thanks.